sprint.go

```diff
@@ -75,8 +85,8 @@ func (s *SprintService) GetIssuesForSprintWithContext(ctx context.Context, sprin
 }
 
 // GetIssuesForSprint wraps GetIssuesForSprintWithContext using the background context.
-func (s *SprintService) GetIssuesForSprint(sprintID int) ([]Issue, *Response, error) {
-	return s.GetIssuesForSprintWithContext(context.Background(), sprintID)
+func (s *SprintService) GetIssuesForSprint(sprintID int, options *GetIssuesForSprintOptions) ([]Issue, *Response, error) {
+	return s.GetIssuesForSprintWithContext(context.Background(), sprintID, options)
```

Comment:
this would become `s.GetIssuesForSprintWithContext(context.Background(), sprintID, encoders...)